frontend/src/insecure/BadHugeComponent.jsx

```
@@ -0,0 +1,49 @@
+    for(let i=0;i<1000000;i++){ s += Math.sqrt(i); }
+    return s;
+  }
+
+  return (
+    <div ref={ref} style={{padding:10, background:"#eee"}}>
+      <h3>BadHugeComponent</h3>
+      <button onClick={()=>setCount(count+1)}>Inc</button>
+      <div>Counter: {count}</div>
+      <div>Calc: {doHeavyStuff()}</div>
+      <ul>
+        {items.map(it => <li>{it.val}</li>)}
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)